sonar-java-plugin/src/main/java/org/sonar/plugins/java/JavaSensor.java

```diff
@@ -185,11 +184,15 @@ private Iterable<InputFile> javaFiles(InputFile.Type type) {
   }
 
   private JavaVersion getJavaVersion() {
```

Comment:
When `settings.get(JavaVersion.ENABLE_PREVIEW)` is empty we should return `new JavaVersionImpl()` without logs